libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat

```diff
@@ -94,6 +94,9 @@ define HAL_GPIO_B_LED_PIN 2
 
 PB4 TIM3_CH1 TIM3 GPIO(58) ALARM LOW
 
+#use LED-STRIP output as general purpose GPIO
+PB6 GP_GPIO OUTPUT LOW GPIO(57)
```

Comment:
we usually use 50 range as PWM pins, maybe use 7x ?

Reply:
done